### PR TITLE
Generate new dev certs [CLOUDDST-10174]

### PR DESCRIPTION
**Pull request by lcarva**

### CODE REVIEW

**Comment by pbortlov** [comment]
:heart: THANK YOU

**Comment by lcarva** [comment]
Travis CI failure is unrelated to these changes.

**Comment by pbortlov** [comment]
updated with thr task: https://issues.redhat.com/browse/CLOUDDST-10174

File: docker-compose.yml [hunk 1]
```diff
@@ -1,21 +1,37 @@
 ---
 version: '3'
 services:
+  minica:
+    image: registry.access.redhat.com/ubi8/go-toolset:latest
```
Comment (pbortlov) [inline docker-compose.yml:5]: Could you add the comments why is mimica needed?

Comment (lcarva) [reply docker-compose.yml:7]: Good idea. Done.

**Review by pbortlov — DISMISSED** [review]

**Comment by pbortlov** [comment]
@lcarva could you please rebase?

**Comment by lcarva** [comment]
> @lcarva could you please rebase?

Done!

**Review by pbortlov — APPROVED** [review]
Tested on my machine

**Review by yashvardhannanavati — APPROVED** [review]

**Pull request merged**